lib/cli/gfx/server-util.js

```diff
@@ -0,0 +1,79 @@
+/** Modified script from blessed-contrib server-util */
+const blessed = require('blessed'),
+      contrib = require('blessed-contrib'),
+      Stream  = require('stream');
+
+function OutputBuffer(options) { 
+  this.isTTY = true;
+  this.columns = options.cols;
+  this.rows = options.rows;
+  this.write = function(s) {
+    s = s.replace('\x1b8', ''); //not clear from where in blessed this code comes from. It forces the terminal to clear and loose existing content.
+    options.stream.write(s)
+  };
+
+  this.on = function() {};
+}
+
+function InputBuffer() {
+  this.isTTY = true;
+  this.isRaw = true;
+  this.emit = function() {};
+  this.setRawMode = function() {};
+  this.resume = function() {};
+  this.pause = function() {};
+  this.on = function() {};
```

Comment:
convert to arrow function if possible so it will be consistent with the other functions below